Check for Accessibility

Yerimah opened the pull request:
## Accessibility Report

In this branch, the following checks were made and optimized accordingly

- Page titles - no issues found
- Image text alternatives [Link to changes made](https://github.com/Yerimah/My-portfolio/blob/643a293213cd00c65e6a6940e9354e2bc9b445d9/index.html#L73-L89) 
[Second link to image text alternative](https://github.com/Yerimah/My-portfolio/blob/643a293213cd00c65e6a6940e9354e2bc9b445d9/index.html#L293-L309)
- Text headings - no issues found
- Color contrast - no issues found [Link to changes made](https://github.com/Yerimah/My-portfolio/blob/ff3fcdfb94380fc96a78b750a2014ca2b6d831de/style.css#L612-L614)
- Resize - no issues found
- Interaction - no issues found
- moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - no issues found

Yerimah committed:
add image text alternatives

SirriRyisa committed:
Add label tags to form for accessibility

Yerimah committed:
modify html and css file to correct linter error

VanessaAoki commented on index.html:
```diff
@@ -70,37 +70,38 @@ <h3 class="supporting-text">I’am a Software Developer</h3>
             <ul class="social-icons">
                 <li>
-                    <a href="#"><img src="./images/github.png" alt=""/></a>
+                    <a href="#"><img src="./images/github.png" alt="github icon"/></a>
```
- [ ] Great job adding the alt text for the icons images 👏  but for functional images, the alt text should convey its purpose, you can read more about it [here](https://www.w3.org/WAI/fundamentals/accessibility-principles/#alternatives). For instance, instead of having `github icon`, you could have `github link`.
- [ ] Beware that this change should be applied to all icons, both here and in the footer.

VanessaAoki commented on index.html:
```diff
@@ -70,37 +70,38 @@ <h3 class="supporting-text">I’am a Software Developer</h3>
-            <img class="w-image" src="./images/placeholder.png" alt=""/>
+            <img class="w-image" src="./images/placeholder.png" alt="first project image"/>
             <div class="right-block">
             <h2>Multi-Post Stories</h2>
```
- [ ] To keep your HTML semantic, you should consider making this heading an `<h3>`. You already have a heading for this whole section, which is an `<h2>`, so hierarchically, this is lower. Also, all the other articles have an `<h3>` heading! You can read more about headings [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)

VanessaAoki reviewed (changes requested):
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @Yerimah and @SirriRyisa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL]: To keep your repo professional, you should consider adding an updated screenshot to your README file.

![so close](https://c.tenor.com/nMbdSNsg_jsAAAAC/the-simpsons-homer-simpson.gif)

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Yerimah committed:
modify html, css and README.md

Implemented changes
- Edit image text alterantive for social icons
- Change the screenshot on my README.md file to an updated one
- Added styling to h3 tag

alonsofalconi reviewed (approved):
Hi @Yerimah and @SirriRyisa,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

![](https://media1.giphy.com/media/d31w24psGYeekCZy/giphy.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
